modules/nf-core/sentieon/bwamem/tests/main.nf.test

```diff
@@ -59,140 +59,182 @@ nextflow_process {
-
-    //     then {
-    //         assertAll(
-    //             { assert process.success },
-    //             { assert snapshot(
-    //                 file(process.out.bam[0][1]).name,
-    //                 file(process.out.csi[0][1]).name,
-    //                 process.out.versions
-    //             ).match() }
-    //         )
-    //     }
-    // }
+    test("Paired-End") {
+        config "./nextflow.config"
+
+        setup {
+            run("SENTIEON_BWAINDEX") {
+                script "../../bwaindex/main.nf"
+                process {
+                    """
+                    input[0] = [
+                        [id: 'test'],
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                    ]
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    [
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_1.fastq.gz', checkIfExists: true),
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_2.fastq.gz', checkIfExists: true)
+                    ]
+                ]
+                input[1] = SENTIEON_BWAINDEX.out.index
+                input[2] = [[id: 'test'],file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)]
+                input[3] = [[id: 'test'],file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta.fai', checkIfExists: true)]
+                """
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    [
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_1.fastq.gz', checkIfExists: true),
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_2.fastq.gz', checkIfExists: true)
+                    ]
+                ]
+                input[1] = SENTIEON_BWAINDEX.out.index
+                input[2] = [[id: 'test'],file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)]
+                input[3] = [[id: 'test'],file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta.fai', checkIfExists: true)]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.bam_and_bai,
+                    process.out.versions,
+                    file(process.out.bam_and_bai[0][1]).name
+                ).match()
```

Comment:
```suggestion
                { assert snapshot(process.out).match()
```
Is there a reason to not just do this?

Reply:
no reason I pretty much just took what was there and extended it

Reply:
there you go